CHANGES.md

```diff
@@ -4,6 +4,10 @@
 
 - Support for OCaml 5.4 (#1559)
 
+## Fixes
+
+- Improve precision of collected metric timestamps. (#1565)
+
```

Comment:
This is not the correct changelog section, you should create a new "unreleased" section.